js/content/community.js

```diff
@@ -330,7 +330,7 @@ let ProfileHomePageClass = (function(){
 
         let iconType = "none";
         let images = SyncedStorage.get("show_profile_link_images");
-        if (images !== false) {
+        if (images !== "false") {
```

Comment:
I choose to not convert it to an actual boolean, since in `options.js` the string was also compared, instead of doing any kind of converting.

Reply:
Could you give me a reference of that?

Reply:
https://github.com/tfedor/AugmentedSteam/blob/develop/js/options/options.js#L418

Reply:
I will change this to `"none"`

Reply:
Changing this to `"none"` would help